views/summer-of-code-2015.slim

```diff
@@ -24,20 +24,20 @@ css:
 
           We're participating in [Rails's Girls Summer of Code 2015](https://github.com/rails-girls-summer-of-code/projects). This is a great chance to work on Open Source and to get paid doing it!
 
-          You'll get to work with [mentors](/about) who have worked on all aspects of the CocoaPods ecosystem, and to improve tooling that positively affects almost every iOS / Mac application.
+          You'll get to work with [mentors](/about#team) who have worked on all aspects of the CocoaPods ecosystem, and to improve tooling that positively affects almost every iOS / Mac application.
 
           We currently have the following projects for the summer:
 
-        - for project in data.summer_of_code_projects do
+        - for project in data("summer_of_code_projects") do
 
-          h3 = project.project_name
-          p = project.project_bio
+          h3 = project[:project_name]
+          p = project[:project_bio]
 
-          ul#projects
-            - for subproject in project.projects
-              li
-                p = "#{subproject.repos} - #{subproject.title}"
-                a
-                  a href="#{subproject.issue_url}" = subproject.issue_url.split("github.com/")[-1].gsub("/issues/", "#")
-                p = subproject.description
-                hr
+          / ul#projects
```

Comment:
This doesn't seem to be related to the main purpose of this PR (Sinatra).

What's the intention here?